definitions/marts/looker_studio_marts/ecf/ls_ecf_correct_cohort_checker.sqlx

```diff
@@ -0,0 +1,241 @@
+            niot_partnership_chk = 0 \n \
+            AND \n \
+            earliest_dec_suggestion_match = 0 \n \
+        For ECTs in the 'wrong' cohort without declarations: \n \
+           WHERE \n \
+            eligible_for_funding = TRUE \n \
+            AND \n \
+            participant_role = 'ECT' \n \
+            AND \n \
+            declaration_id IS NULL \n \
+            AND \n \
+            has_relevant_declarations = FALSE \n \
+            AND \n \
+            -- Check if the lead provider is NIOT and there is a Live Partnership in place. \n \
+            niot_partnership_chk = 0 \n \
+            AND \n \
+            start_date_suggestion_match = 0 \n \
+    Dependencies: \n \
+        dataform.ecf_declarations",
+    columns: {
+        user_id: "This comes from the teacher profile associated with the participant profile. User ID also exists independently in other tables.",
+        participant_profile_id: "The ID from a participant's profile in the CPD service.",
+        participant_role: "The role of the participant which is either ECT or Mentor as per the participant type.",
+        current_schedule: "The schedule as per the participant profile.",
+        declaration_id: "The unique identifier for the declaration.",
+        declaration_date: "The date of when evidence was submitted against the declaration.",
+        lead_provider_name: "The name of the Lead Provider that submitted the declaration.",
+        induction_lead_provider_name: "The name of the Lead Provider associated with the induction record. Used for ECTs & Mentors with zero s/e/p/p declarations.",
+        declaration_type: "The type of declaration which can be started, retained-#, extended-# or completed. #'s range from 1 to 4.",
+        declaration_state: "The state of the declaration which for this mart can only be submitted, eligible, payable, or paid. (s/e/p/p)",
+        current_cohort: "The cohort/academic year corresponding to when the participant started their course as per the service.",
+        induction_start_date: "The date when the ECT or Mentor started their induction taken from their Participant Profile.",
+        eligible_for_funding: "TRUE/FALSE flag for funding eligibility.",
+        partnership_id: "ID for the Lead Provider-School partnership for a given cohort.",
+        partnership_relationship: "TRUE/FALSE for the partnership_id taken from partnerships_latest_cpd. The relationship is 'live' when the value returns FALSE.",
+        earliest_declaration_date: "The first declaration date for the participant_profile_id where the declaration is funded (eligible, payable or paid).",
+        unique_participant_counter: "1 or 0 if the row is the first instance of the participant_profile_id. Used in reporting as a means of gaining unique count of participants.",
+        niot_partnership_chk: "1 or 0 if the declaration relates to a participant with an active school-lead provider relationship and the lead provider is NIOT",

```

Comment:
Can you clarify the policy reason this field exists? I don't understand why it's being included from this description alone.

Reply:
I'm still confused, why are these cases excluded? What is the policy connection?

Reply:
I think it should be in the description in the table, why do we permit these discrepancies?